docs/concepts/how-to-grant-api-access.md

```diff
@@ -0,0 +1,181 @@
+
+# Granting Users Access to API Resources in OLM
+
+When operators or cluster extensions are managed via OLM, they often provide Custom Resource Definitions (CRDs) that expose new API resources. Typically, cluster administrators hold full management access to these resources by default, whereas non-administrative users may lack sufficient permissions. Such users often need designated permissions to create, view, or edit these Custom Resources.
```

Comment:
```suggestion
When ClusterExtension are managed via OLM, they often provide Custom Resource Definitions (CRDs) that expose new API resources. Typically, cluster administrators hold full management access to these resources by default, whereas non-administrative users may lack sufficient permissions. Such users often need designated permissions to create, view, or edit these Custom Resources.
```

Reply:
The meeting convey seems to keep ClusterExtension only.

Reply:
yep, changed all refs from `operator` to `cluster extension`.